### ISSUE TITLE
Public List of Data Sets

### ISSUE THREAD

**Issue by deiruch** [i4Ds/Karabo-Pipeline#285]
Slava asked for a list of data sets. His especially interested in real datasets, useful for ML.

We could start such a list, as part of our documentation. Mark Sargent's talk in the 2023-01-27 meeting includes a number of datasets. Just a table with links, and short descriptions would be interesting.

**Comment by Lukas113** [comment]
I don't think this should be part of Karabo? For data sets we already provide, descriptions should exist If such a request exists (either in README or in API doc).

I think such requests should be handled separately of Karabo dev efforts because it doesn't affect Karabo itself.

**Comment by sfiruch** [comment]
Agree!. But we should at least have this table in the Karabo documentation.
